src/components/v5/common/Fields/Input/Input.tsx

```diff
@@ -106,7 +111,7 @@ const Input: FC<InputProps> = ({
       )}
 
       {/* This is to stop layout shift when error messages are shown */}
-      <div className="relative pb-9">
+      <div className="relative pb-[1.125rem]">
```

Comment:
Are these 2px very important @melyndav? Otherwise I would recommend using `pb-4` (16px) or `pb-5` (20px). This helps us to stay consistent across all components.

Reply:
@chmanie do you have a screenshot of the UI for this? It's hard to make a call on the padding change when I don't know what I'm looking. 👍